core/src/main/scala/org/elasticmq/util/NowProviderHolder.scala

```diff
@@ -0,0 +1,6 @@
+package org.elasticmq.util
+
+trait NowProviderHolder {
+
+  val nowProvider: NowProvider = new NowProvider
```

Comment:
Maybe it would be better to create one abstraction without any behaviour:
```scala
trait NowProviderHolder {
    def nowProvider: NowProvider
}
```

and then instantiate two independent concrete implementations:
```scala
trait MutableNowProviderHolder extends NowProviderHolder { ... }
trait RealNowProviderHolder extends NowProviderHolder { ... }
```

Reply:
I've removed holders, NowProviders are passed explicitly